content/en/events/upcoming-events/gsoc-proposal-template.md

```diff
@@ -0,0 +1,66 @@
+# Project Title
+
+[Provide a exact same title for your project from the proposals page; including the project number as is]
+
+## Summary
+
+[Offer a brief summary of your project, highlighting the problem it addresses, your proposed solution, and the anticipated impact.]
+
+## Personal Information
+
+- **Full Name**: [Your Full Name]
+- **Email Address**: [Your Email Address]
+- **GitHub Profile**: [Your GitHub Profile URL]
+- **LinkedIn**: [Your LinkedIn Profile URL]
+- **University/College**: [Your University/College Name]
+- **Degree Program**: [Your Degree Program]
+- **Year of Study**: [Your Current Year of Study]
+- **Country of Residence**: [Your Country]
+
+## Qualifications; Why Me?
+
+[Highlight your relevant skills, experiences, and qualifications that make you a suitable candidate for this project. Mention any previous work, open-source contributions, or academic achievements that showcase your capabilities. List any/all contributions to Kubeflow community so far.]
+
+[Be clear about your availability and time commitment. State the number of hours per week you plan to dedicate to the project (recommended: 30-40 hours). Mention any potential conflicts (e.g., exams, vacations) and how you plan to address them. It is important to manage expectations upfront and explain how you will manage your time effectively to meet the project deadlines.]
+
+## Goals
+
+[List the specific goals you aim to achieve during the GSoC coding period. Be clear and concise. Example: Implement feature X in library Y., Example: Improve the performance of module Z by N%.]
+
+## Non-Goals
+
+[Clearly state what is *out* of scope for this project. This helps manage expectations. Example: This project will not include integration with service A., Example: We will not be addressing issue B during this GSoC project.]
+
+## Technical Details
+
+[Provide an in-depth description of your project's technical aspects. This should include the technologies, programming languages, frameworks, and tools you plan to use. Discuss any potential challenges and how you intend to address them.]
+
+## Test Plan
+
+[Describe how you plan to test your code to ensure its quality and correctness.]
+
+## Schedule of Deliverables
+
+[Present a rough timeline of your project, breaking down each phase with specific tasks and deadlines. This schedule should align with the GSoC timeline and demonstrate a clear plan for project completion.]
```

Comment:
I am a little concerned that students will attempt to create a prescriptive plan which is unlikely to actually be correct.

We should probably just have the student focus on breaking the goals down into stages, and estimating how long they will take, relative to the allowable period.

Reply:
yes, I agree. since the solution was not accepted yet, discussed.

Reply:
ok, I will remove any specific dates; I was aligning them with mid-terms and finals